<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PulsedNMR</class>
 <widget class="QMainWindow" name="PulsedNMR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PulsedNMR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitterH">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Signal</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="PlotTimeWidget" name="plotTimeWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
           </widget>
           <widget class="PlotFFTWidget" name="plotFFTWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="controlWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="acquisitionGroup">
           <property name="title">
            <string>Acquisition</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="3" column="0">
             <widget class="QLabel" name="rxTimeLabel">
              <property name="text">
               <string>Acquisition time</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="rateValue"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="rxSamplesLabel">
              <property name="text">
               <string>Acquisition samples</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="rateLabel">
              <property name="text">
               <string>Sample rate</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QDoubleSpinBox" name="averageValue">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="suffix">
               <string extracomment="Measurement Average Count"/>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>20.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="averageLabel">
              <property name="text">
               <string>Averaging</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QCheckBox" name="averageCheck">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>enabled</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="rxTimeValue">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string> ms</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.500000000000000</double>
              </property>
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="rxSamplesEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="mode0Group">
           <property name="title">
            <string>Mode 0: Search</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QCheckBox" name="plotTimeCheck">
              <property name="text">
               <string>Plot Time Signal</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="plotFFTCheck">
              <property name="text">
               <string>Plot FFT</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="plotTimeModeDrop">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Amplitude</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>In-Phase</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>I+Q</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="mode1Group">
           <property name="title">
            <string>Mode 1: Half-Time</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="freqLlabel">
              <property name="text">
               <string>Frequency</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="awidthValue">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string extracomment="periods"> us</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="awidthLabel">
              <property name="text">
               <string>A width</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="freqValue">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string> MHz</string>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>50.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.001000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="deltaValue">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> ms</string>
              </property>
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="singleStep">
               <number>20</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="deltaLabel">
              <property name="text">
               <string>A-to-A delay</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="mode2Group">
           <property name="title">
            <string>Mode 2: Echo</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="bwidthLabel">
              <property name="text">
               <string>B width</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="d1widthLabel">
              <property name="text">
               <string>A-B delay</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="bwidthValue">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string extracomment="periods"> us</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>200.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="bcountLabel">
              <property name="text">
               <string>B count</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="d1widthValue">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string extracomment="periods"> ms</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.500000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="bcountValue">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string extracomment="periods"/>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="bbdelayLabel">
              <property name="text">
               <string>B-to-B delay</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="bbdelayValue">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string extracomment="periods"> ms</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.500000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>109</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="LogWidget" name="logWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="menuConnect"/>
    <addaction name="separator"/>
    <addaction name="menuExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menuExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menuConnect">
   <property name="text">
    <string>Connect..</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogWidget</class>
   <extends>QWidget</extends>
   <header>logwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotTimeWidget</class>
   <extends>QWidget</extends>
   <header>plotwidgets.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotFFTWidget</class>
   <extends>QWidget</extends>
   <header>plotwidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
